content/python/concepts/data-visualization/terms/line/line.md

```diff
@@ -0,0 +1,44 @@
+# `.line()` Method in Plotly
+
+## Introduction
+
+The `.line()` method in Plotly is specifically designed for creating line charts, which effectively illustrate changes and trends over time. Line charts are invaluable in data visualization for depicting the progression of numerical data points connected by line segments. This method simplifies the visualization of relationships between elements in datasets, highlighting trends and changes across continuous or categorical axes.
+
+## Syntax
+
+The syntax for the `.line()` method in Plotly is straightforward and highly customizable. Here is a general structure:
+
+```python
+plotly.express.line(data_frame, x, y, color=None, title=None, labels=None, ...)
+```
+- data_frame: The dataset containing the variables that will be plotted.
+- x: The data for the x-axis, typically time or a continuous variable.
+- y: The data for the y-axis, representing the measured variables.
+- color: (Optional) A column name in the dataframe that will determine the color of the lines, useful for distinguishing different groups or categories.
+- title: (Optional) The title of the chart.
+labels: (Optional) Custom labels for different axes or legend items.
+
+```
+
+## Example
+
+Below is an example demonstrating how to use the `.line()` method to create a line chart visualizing economic growth over time:
+
+```python
+import plotly.express as px
+
+# Load an example dataset
+df = px.data.gapminder().query("continent == 'Oceania'")
+
+# Create a line chart
+fig = px.line(df, x='year', y='gdpPercap', color='country', title='Economic Growth in Oceania')
+
+# Show the plot
+fig.show()
+
+```
+
```

Comment:
Please give the Output under the ## Output section. You can attach the image of the output using the Image block.
Please refer to the [Style Guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) to add the output Image.

Reply:
Done!